src/styles/main.scss

```diff
@@ -0,0 +1,11 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  background: linear-gradient(12.3deg, rgb(129, 238, 129),
+  rgb(255, 217, 0), rgb(121, 21, 21));
+}
+
+@import "./watch.scss";
```

Comment:
It's a good practice to import partials containing keyframes and other global styles first.